### DISCUSSION TITLE
default colorscheme overrides terminal colors for :Man

### DISCUSSION THREAD

**Discussion by sarmong** [neovim/neovim#28716] [General]
### Problem

On 0.10, when you use neovim as a manpager without setting your own scheme, the new default colorscheme will be used.

### Steps to reproduce

Nightly:

```bash
VIMRUNTIME="runtime" MANPAGER="./build/bin/nvim --clean -u NONE +Man!" man man
```

![image](https://github.com/neovim/neovim/assets/42828375/94922f5c-0e32-46ff-a87f-ff507c8958fe)

0.9.5:

```bash
MANPAGER="nvim --clean -u NONE +Man!" man man
```

![image](https://github.com/neovim/neovim/assets/42828375/40aed87a-dae2-4725-b95c-c0f18b3dd805)

### Expected behavior

Default colorscheme should not be used and terminal colors should be applied as on 0.9.5

### Neovim version (nvim -v)

v0.10.0-dev-3117+g4e5c633ed

### Vim (not Nvim) behaves the same?

-

### Operating system/version

Linux/X11

### Terminal name/version

wezterm

### $TERM environment variable

xterm-256color

### Installation

Build from repo

**Comment by clason** [comment]
That is expected, is it not? The default colorscheme is always used if you don't set your own; 0.10 just has a new (more minimal) colorscheme. If you prefer the old one, set `:colorscheme vim` in your config.

**Comment by sarmong** [comment]
But when I use neovim as a manpager, I want none of my config to load. 

IMHO, the default colorscheme doesn't make sense for a manpager, as it doesn't have any highlighting.

**Comment by clason** [comment]
Sorry, but that is just an egg we had to break to make the omelette, then. You don't need to add the full config, just `+"colorscheme vim"` to the `MANPAGER` invocation.

**Comment by justinmk** [comment]
> just `+"colorscheme vim"` to the `MANPAGER` invocation.

Or `+"set notermguicolors"` , if the request here is to use ANSI colors?

**Reply by sarmong** [reply]
This doesn't add the same highlighting on its own in default scheme
